code/datums/status_effects/debuffs/staggered.dm

```diff
@@ -43,3 +46,13 @@
 /mob/living/proc/do_stagger_animation()
 	animate(src, pixel_x = 4, time = 0.2 SECONDS, loop = 6, flags = ANIMATION_RELATIVE|ANIMATION_PARALLEL)
 	animate(pixel_x = -4, time = 0.2 SECONDS, flags = ANIMATION_RELATIVE)
+
+/// Status effect specifically for instances where shoving someone has resulted in them falling over/onto an object
+
+/datum/status_effect/shoved_over
```

Comment:
Can you rename this status effect to something more specific, like "next_shove_stuns" or something? Because like I said I want us to end up reintroducing this more piecemeal in the future

Reply:
I tried to be more specific but yeah no problem.